### PR TITLE
Generate Reliably CI/CD workflow

### PR DESCRIPTION
**Pull request by dmartin35**

### CODE REVIEW

**Review by xaah — APPROVED** [review]
Looks good!

**Review by mperrien — CHANGES_REQUESTED** [review]
The content is great, and I really like the CLI flow.

I just made a few wording corrections.

File: docs/getting-started/ci-workflow/index.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+```console
+$ reliably workflow
+```
+
+You will then be prompted for choosing one of the targeted platform,
+for which we can generate a default workflow template.
+
+```console
+? Select your CI/CD platform to create the Reliably workflow ?  [Use arrows to move, type to filter]
```
[the remark this answers is not in the document]
Comment (dmartin35) [reply docs/getting-started/ci-workflow/index.md]: Good remark, I updated the doc, and I'll make the change in the CLI as well
However , I cannot removes the leading question mark, it's rendered by the interactive library that we're using

**Comment by dmartin35** [comment]
FYI, I have made the changes @mperrien

**Pull request merged**